.spi.yml

```
@@ -0,0 +1,4 @@
+version: 1
+builder:
+  configs:
+    - documentation_targets: [SystemPackage]
```

nit: missing newline
```suggestion
    - documentation_targets: [SystemPackage]

```